packages/twenty-server/src/engine/api/graphql/workspace-schema-builder/factories/composite-input-type-definition.factory.ts

```diff
@@ -62,6 +62,14 @@ export class CompositeInputTypeDefinitionFactory {
         continue;
       }
 
+      // Skip secondaryLinks property for LINKS type in Filter input type
+      if (
```

Comment:
can we actually try to change packages/twenty-server/src/engine/metadata-modules/field-metadata/composite-types/links.composite-type.ts to have hidden: 'input' on secondaryLinks. This should leverage the continue check line 61 of composite-input-type-definition.factory and avoid adding this extra edge case